
Cleanup pkg-plist a bit to get correct package w/ different options

--- pkg-plist.orig	2015-12-16 09:02:22.000000000 +0200
+++ pkg-plist
@@ -88,7 +88,7 @@ share/applications/blender.desktop
 %%OPENCOLORIO%%%%DATADIR%%/%%VER%%/datafiles/colormanagement/luts/spi_ocio_srgb_test.spi3d
 %%OPENCOLORIO%%%%DATADIR%%/%%VER%%/datafiles/colormanagement/luts/srgb.spi1d
 %%OPENCOLORIO%%%%DATADIR%%/%%VER%%/datafiles/colormanagement/luts/srgb_to_xyz.spimtx
-%%NLS%%%%DATADIR%%/%%VER%%/datafiles/colormanagement/luts/vd16.spi1d
+%%OPENCOLORIO%%%%DATADIR%%/%%VER%%/datafiles/colormanagement/luts/vd16.spi1d
 %%NLS%%%%DATADIR%%/%%VER%%/datafiles/fonts/bmonofont-i18n.ttf.gz
 %%NLS%%%%DATADIR%%/%%VER%%/datafiles/fonts/droidsans.ttf.gz
 %%NLS%%%%DATADIR%%/%%VER%%/datafiles/locale/ar/LC_MESSAGES/blender.mo
@@ -219,6 +219,7 @@ share/applications/blender.desktop
 %%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_montecarlo.h
 %%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_passes.h
 %%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_path.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_path_branched.h
 %%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_path_common.h
 %%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_path_state.h
 %%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/kernel_path_surface.h
@@ -295,6 +296,7 @@ share/applications/blender.desktop
 %%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_value.h
 %%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_vector_transform.h
 %%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_voronoi.h
+%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_voxel.h
 %%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_wave.h
 %%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_wavelength.h
 %%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/kernel/svm/svm_wireframe.h
@@ -314,89 +316,89 @@ share/applications/blender.desktop
 %%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/osl.py
 %%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/presets.py
 %%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/properties.py
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_absorption_volume.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_add_closure.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_ambient_occlusion.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_anisotropic_bsdf.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_attribute.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_background.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_blackbody.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_brick_texture.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_brightness.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_bump.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_camera.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_checker_texture.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_color.h
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_combine_hsv.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_combine_rgb.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_combine_xyz.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_convert_from_color.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_convert_from_float.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_convert_from_int.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_convert_from_normal.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_convert_from_point.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_convert_from_vector.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_diffuse_bsdf.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_emission.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_environment_texture.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_fresnel.h
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_fresnel.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_gamma.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_geometry.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_glass_bsdf.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_glossy_bsdf.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_gradient_texture.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_hair_bsdf.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_hair_info.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_holdout.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_hsv.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_image_texture.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_invert.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_layer_weight.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_light_falloff.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_light_path.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_magic_texture.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_mapping.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_math.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_mix.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_mix_closure.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_musgrave_texture.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_noise_texture.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_normal.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_normal_map.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_object_info.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_output_displacement.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_output_surface.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_output_volume.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_particle_info.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_refraction_bsdf.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_rgb_curves.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_rgb_ramp.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_scatter_volume.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_separate_hsv.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_separate_rgb.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_separate_xyz.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_set_normal.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_sky_texture.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_subsurface_scattering.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_tangent.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_texture.h
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_texture_coordinate.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_toon_bsdf.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_translucent_bsdf.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_transparent_bsdf.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_uv_map.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_value.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_vector_curves.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_vector_math.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_vector_transform.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_velvet_bsdf.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_voronoi_texture.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_wave_texture.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_wavelength.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_wireframe.oso
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/oslutil.h
-%%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/stdosl.h
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_absorption_volume.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_add_closure.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_ambient_occlusion.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_anisotropic_bsdf.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_attribute.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_background.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_blackbody.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_brick_texture.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_brightness.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_bump.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_camera.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_checker_texture.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_color.h
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_combine_hsv.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_combine_rgb.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_combine_xyz.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_convert_from_color.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_convert_from_float.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_convert_from_int.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_convert_from_normal.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_convert_from_point.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_convert_from_vector.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_diffuse_bsdf.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_emission.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_environment_texture.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_fresnel.h
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_fresnel.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_gamma.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_geometry.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_glass_bsdf.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_glossy_bsdf.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_gradient_texture.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_hair_bsdf.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_hair_info.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_holdout.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_hsv.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_image_texture.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_invert.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_layer_weight.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_light_falloff.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_light_path.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_magic_texture.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_mapping.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_math.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_mix.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_mix_closure.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_musgrave_texture.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_noise_texture.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_normal.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_normal_map.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_object_info.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_output_displacement.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_output_surface.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_output_volume.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_particle_info.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_refraction_bsdf.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_rgb_curves.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_rgb_ramp.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_scatter_volume.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_separate_hsv.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_separate_rgb.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_separate_xyz.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_set_normal.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_sky_texture.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_subsurface_scattering.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_tangent.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_texture.h
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_texture_coordinate.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_toon_bsdf.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_translucent_bsdf.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_transparent_bsdf.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_uv_map.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_value.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_vector_curves.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_vector_math.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_vector_transform.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_velvet_bsdf.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_voronoi_texture.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_wave_texture.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_wavelength.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/node_wireframe.oso
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/oslutil.h
+%%CYCLESOSL%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/shader/stdosl.h
 %%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/ui.py
 %%CYCLES%%%%DATADIR%%/%%VER%%/scripts/addons/cycles/version_update.py
 %%DATADIR%%/%%VER%%/scripts/addons/development_api_navigator.py
@@ -744,52 +746,53 @@ share/applications/blender.desktop
 %%DATADIR%%/%%VER%%/scripts/addons/ui_translate/update_ui.py
 %%DATADIR%%/%%VER%%/scripts/addons/uv_bake_texture_to_vcols.py
 %%DATADIR%%/%%VER%%/scripts/addons/uv_texture_atlas.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/modules/freestyle/__init__.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/modules/freestyle/chainingiterators.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/modules/freestyle/functions.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/modules/freestyle/predicates.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/modules/freestyle/shaders.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/modules/freestyle/types.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/modules/freestyle/utils.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/modules/parameter_editor.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/anisotropic_diffusion.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/apriori_and_causal_density.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/apriori_density.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/backbone_stretcher.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/blueprint_circles.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/blueprint_ellipses.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/blueprint_squares.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/cartoon.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/contour.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/curvature2d.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/external_contour.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/external_contour_sketchy.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/external_contour_smooth.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/haloing.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/ignore_small_occlusions.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/invisible_lines.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/japanese_bigbrush.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/long_anisotropically_dense.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/multiple_parameterization.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/nature.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/near_lines.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/occluded_by_specific_object.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/polygonalize.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/qi0.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/qi0_not_external_contour.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/qi1.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/qi2.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/sequentialsplit_sketchy.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/sketchy_multiple_parameterization.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/sketchy_topology_broken.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/sketchy_topology_preserved.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/split_at_highest_2d_curvatures.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/split_at_tvertices.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/suggestive.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/thickness_fof_depth_discontinuity.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/tipremover.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/tvertex_remover.py
-%%DATADIR%%/%%VER%%/scripts/freestyle/styles/uniformpruning_zsort.py
+%%FREESTYLE%%@dir %%DATADIR%%/%%VER%%/scripts/freestyle
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/modules/freestyle/__init__.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/modules/freestyle/chainingiterators.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/modules/freestyle/functions.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/modules/freestyle/predicates.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/modules/freestyle/shaders.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/modules/freestyle/types.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/modules/freestyle/utils.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/modules/parameter_editor.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/anisotropic_diffusion.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/apriori_and_causal_density.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/apriori_density.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/backbone_stretcher.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/blueprint_circles.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/blueprint_ellipses.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/blueprint_squares.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/cartoon.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/contour.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/curvature2d.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/external_contour.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/external_contour_sketchy.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/external_contour_smooth.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/haloing.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/ignore_small_occlusions.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/invisible_lines.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/japanese_bigbrush.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/long_anisotropically_dense.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/multiple_parameterization.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/nature.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/near_lines.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/occluded_by_specific_object.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/polygonalize.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/qi0.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/qi0_not_external_contour.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/qi1.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/qi2.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/sequentialsplit_sketchy.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/sketchy_multiple_parameterization.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/sketchy_topology_broken.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/sketchy_topology_preserved.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/split_at_highest_2d_curvatures.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/split_at_tvertices.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/suggestive.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/thickness_fof_depth_discontinuity.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/tipremover.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/tvertex_remover.py
+%%FREESTYLE%%%%DATADIR%%/%%VER%%/scripts/freestyle/styles/uniformpruning_zsort.py
 %%DATADIR%%/%%VER%%/scripts/modules/addon_utils.py
 %%DATADIR%%/%%VER%%/scripts/modules/animsys_refactor.py
 %%DATADIR%%/%%VER%%/scripts/modules/bl_i18n_utils/__init__.py
@@ -1110,7 +1113,7 @@ share/applications/blender.desktop
 %%DATADIR%%/%%VER%%/scripts/templates_py/ui_previews_custom_icon.py
 %%DATADIR%%/%%VER%%/scripts/templates_py/ui_previews_dynamic_enum.py
 %%PORTDOCS%%%%DOCSDIR%%/GPL-license.txt
-%%NLS%%%%PORTDOCS%%%%DOCSDIR%%/LICENSE-bfont.ttf.txt
+%%PORTDOCS%%%%DOCSDIR%%/LICENSE-bfont.ttf.txt
 %%NLS%%%%PORTDOCS%%%%DOCSDIR%%/LICENSE-bmonofont-i18n.ttf.txt
 %%NLS%%%%PORTDOCS%%%%DOCSDIR%%/LICENSE-droidsans.ttf.txt
 %%PORTDOCS%%%%DOCSDIR%%/Python-license.txt
